src/script/descriptor.cpp

```diff
@@ -2104,7 +2120,7 @@ struct KeyParser {
         return key;
     }
 
-    std::optional<std::string> ToString(const Key& key) const
+    std::string ToString(const Key& key, bool* has_priv_key = nullptr) const
```

Comment:
In https://github.com/bitcoin/bitcoin/commit/deed439bff638abfc298a3b9c4f3d9f75fa18244 "descriptor: ToPrivateString() pass if at least 1 priv key exists"

Nit: 
```diff
diff --git a/src/script/descriptor.cpp b/src/script/descriptor.cpp
index b6539ab12e..028a7299e5 100644
--- a/src/script/descriptor.cpp
+++ b/src/script/descriptor.cpp
@@ -2120,7 +2120,7 @@ struct KeyParser {
         return key;
     }
 
-    std::string ToString(const Key& key, bool* has_priv_key = nullptr) const
+    std::string ToString(const Key& key, bool*) const
     {
         return m_keys.at(key).at(0)->ToString();
     }
```